benchexec/tablegenerator/react-table/src/tests/__snapshots__/Summary.test.js.snap

```diff
@@ -59000,7 +59000,7 @@ exports[`Render Summary for simple-table-with-scaling.table.html 1`] = `
         <div
          className="cell"
          dangerouslySetInnerHTML={"11000000000"}
-         title="avg: 2200000000, max: 2300000000, median: 2200000000, min: 2100000000, stdev: 72000000"
+         title="avg: 2200000000, max: 2300000000, median: 2200000000, min: 2100000000, stdev: 63000000"
```

Comment:
@PhilippWendler I'm a bit confused about this change. Let's go over this and the other topics in our call later

Reply:
The correct (non-rounded) raw values are now added to the JSON on `master`, so this should be fixed after merging.